core/shared/src/main/scala/zio/prelude/fx/ZPure.scala

```diff
@@ -1277,3 +1297,16 @@ trait ZPureLowPriorityImplicits {
         ZPure.tupledPar(fa, fb)
     }
 }
+
+trait ZPureSyntax {
+
+  implicit final class ZPureRefineToOrDieOps[W, S1, S2, R, E <: Throwable, A](self: ZPure[W, S1, S2, R, E, A]) {
```

Comment:
Could this be moved to ZPure trait as a regular method?

Reply:
I don't think so because we want `E` to appear as the upper bound in `E1 <: E` which is inconsistent with a covariant type parameter.

Reply:
Why not just define in companion object instead of trait so will always automatically be in implicit scope?

Reply:
Oh, of course! Good idea :+1: